appveyor.yml

```
@@ -6,21 +6,23 @@ environment:
     - PYTHON: "C:\\Miniconda36-x64"
```

```suggestion
    - PYTHON: "C:\\Miniconda3-x64"
```

Can you make this change really quick, @mattwthompson so we can trigger the appveyor build again.